### PR TITLE
Add pylintfileheader and configure to check for license header issues

### PR DESCRIPTION
**Pull request by levbishop**
This catches all 93 issues fixed in #2424 #2425 #2426 and passes once those are merged

### CODE REVIEW

**Comment by 1ucian0** [comment]
#2424 #2425 #2426 merged!

File: .pylintrc [hunk 1]
```diff
@@ -21,7 +21,10 @@ persistent=yes
 # List of plugins (as comma separated values of python modules names) to load,
 # usually to register additional checkers.
 load-plugins=pylint.extensions.docparams, # enable checking of docstring args
-    pylint.extensions.docstyle  # basic docstring stle checks
+    pylint.extensions.docstyle,  # basic docstring stle checks
```
Comment (lcapelluto) [inline .pylintrc]: If we're touching this line anyway, could we fix the typo? 😊 
```suggestion
    pylint.extensions.docstyle,  # basic docstring style checks
```

**Comment by 1ucian0** [comment]
Thanks Lev!

File: .pylintrc (hunk 1 above)
Comment (1ucian0) [reply .pylintrc]: I have no permission to touch this branch, so I fix it here f4a64d47. Thanks!

**Pull request merged**